### PR TITLE
Bug 1880345: Provide more details for deployment pods

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -147,6 +148,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	operatorCtx.resourceSyncController = resourceSyncer
 	operatorCtx.operatorClient = operatorClient
 	operatorCtx.operatorConfigInformer = configinformer.NewSharedInformerFactoryWithOptions(configClient, resync)
+	operatorCtx.kubeInformers = informers.NewSharedInformerFactory(operatorCtx.kubeClient, 10*time.Minute)
```
Comment (sttts) [inline pkg/operator/starter.go:151]: why do we need kubeInformersForNamespaces and kubeInformers ?

Comment (sttts) [reply pkg/operator/starter.go:151]: be sure that you don't watch pods for all namespaces.

**Comment by mfojtik** [comment]
/hold

the informers needs to be reconsidered

**Pull request merged**